### ISSUE TITLE
CoLab rholang, rnode office hours

### ISSUE THREAD

**Issue by David405** [rchain/bounties#988]
The Cooperation Laboratory starting in [September](https://docs.google.com/document/d/1BgtKpnwAYQNDN7MWpubrKpiIqerTpYWVAk3ehOvxSyo/edit#) hosts classes and workstudy for rholang on tuesdays and then node setup and testing on thursdays. The venue for the office hours is https://zoom.us/j/6853551826, time is 4pm NY time, 8pm GMT

### Benefit to RChain

- Create an environment for noobs and developers seeking to learn the rholang programming language.

- Provide workstudy for rholang codes, contracts to be evaluated, debugged.

- Guide people through setting up rnode 

- Create our own private RChain network

_Estimated Budget of Task:_ See [voting sheet](https://docs.google.com/spreadsheets/d/1o_-ogVx_tpLhFUNvWq6oex36di8EWK28obnV9J8mJMc/edit#gid=1270017294)
_Estimated Timeline Required to Complete the Task:_ **[weekly]**
_How will we measure completion?_ [[log note]](https://docs.google.com/document/d/1HNIq_sBupkrtO3yuZT_5OlBWw8UEM3O9vNeQGtq9dpw/edit#)

*See CONTRIBUTING.md for details on budget and reward process.*

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by JoshOrndorff** [comment]
I've spoken with @jeremybeal11 about a similar idea. This could become an officially sponsored task, but we are blocked by:

1. Is that budget for each hour? Each week? Each month? Does it go only to pre-selected coaches, or also learners? What if people who intended to be learners actually contribute valuable teaching?
2. Success can't just be offering the event. We need something like at least 5 parties attend each week of the month.

**Comment by jeremybeal11** [comment]
just saw this, and im also curious about what the budget is for. just a side note, can we refrain from using "Noob" and substitute it with "beginning learners" or something like that lol

**Comment by dckc** [comment]
I argued against "noob" long ago too. @jimscarver agreed to some extent, but he keeps using the term.

**Comment by dckc** [comment]
Regarding **[weekly]**, indefinite durations are awkward, so it's important to give plenty of justification. See, for example, the "How will we measure completion?" section of #208.

**Comment by dckc** [comment]
Regarding rewards for meetings, advice I have received that seems wise is: reward for the results of the meetings, not just for holding meetings.

So perhaps the goals here is something like: present the rholang tutorial to 15+ developers in the next month or two or three. Or: develop rholang proficiency in 15+ developers (as demonstrated by the ability to answer the quiz questions in the tutorial).

And for rnode office hours: assist 10+ members in bringing up nodes, or resolve 20+ support issues raised by rnode operators, or some such.

**Comment by JoshOrndorff** [comment]
Yeah, actually, these sessions might be a great chance for a working group to go through the tutorial and solve the exercises and point out where I've made mistakes writing them. Thoughts about merging this in with #919 ?

**Comment by dckc** [comment]
Maybe use #919 for rholang stuff and re-scope this one to rnode stuff?

**Comment by JoshOrndorff** [comment]
Here's my proposal. @jimscarver Does this sound reasonable?

It is valuable to offer peer learning sessions at various times and at various skill levels. I want to close this issue and open two new issues. One for colab rholang sessions and one for colab rnode sessions. The offered budget would be $100 per session and sessions offered once per week.

For each session someone from colab would need to post this template in the bounty issue (not a google doc or an email or a different bounty or a post-it-note) but directly in the corresponding bounty.

Date: 
Attendees:
Topics covered:
Recording video:

It would be hard to argue more than $100 per hour-long session because that's more than my rholang office hours cost the coop. I'll leave it to someone else to create the individual issues and add the developer education label. If this happens I will fund it.

**Comment by David405** [comment]
@joshorndoff what about I change the name of this issue to just rholang office hours and then create a new issue for rnode office hours instead of opening two new issues?

**Comment by JoshOrndorff** [comment]
Actually, @David405, Let's do it the other way around. You start a new issue for rholang office hours according to the details I DMed you on discord.

cc @Eknir

**Comment by JoshOrndorff** [comment]
Let's team up with @ddayan to have this cover his Tuesday morning sessions as well.
